### ISSUE TITLE
[Templates] can we chose between appsettings and parameters?

### ISSUE THREAD

**Issue by Viktorh89** [Azure/LogicAppsUX#5475]
### Describe the Bug with repro steps

Instead of getting all env-specific values as params, is it possible to support appsettings as well?

one reason (there are many) for this is, how do we refer ex KeyVault here without avoiding
params.json:

    "queueName_secret": {
        "type": "securestring",
        "value": "@appsetting('secretQueuename')"
    },

maybe this is possible or solvable in other ways - if so please let me know

### What type of Logic App Is this happening in?

Standard (Portal)

### Which operating system are you using?

Windows

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Browser

chrome

**Comment by zzznz27** [comment]
You can reference app settings directly in logic app using `@appsetting('appsettingName')`
See here: https://learn.microsoft.com/en-us/azure/logic-apps/create-parameters-workflows?tabs=consumption#parameters-for-consumption-versus-standard-logic-app-workflows

**Comment by preetriti1** [comment]
@divyaswarnkar - It seems the customer here want capability to choose the appsetting in the parameters tab while authoring.

@zzznz27 - Can you confirm that is what you want or you want a toggle in each parameter that when you put the value it goes in appsetting instead of parameters.json?

FOr now you can always put the expression and go ahead with creation given you already have the appsetting saved.

**Comment by Viktorh89** [comment]
@preetriti1  no, i want to use @appsettings instead of params ideally (not defaulting to params)
